templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -358,8 +358,6 @@
           }
         }
 
-        var url = {/literal}{$dataUrl|@json_encode}{literal};
```

Comment:
I can't see this javascript var being used at all later on in this JS function

Reply:
Yep, checks out: https://github.com/civicrm/civicrm-core/commit/fea7809f43851b8900d048ae13ac85c0de9149c6